scripts/meetingbaas_api.py

```
@@ -195,9 +246,10 @@ def create_meeting_bot(
 
         response = requests.post(url, json=config, headers=headers)
```

Ruff (0.15.11)

[error] Probable use of `requests` call without timeout

(S113)